src/saltext/vmware/states/storage_policy.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: Apache-2.0
+import logging
+
+import saltext.vmware.utils.connect as connect
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = "vsphere_storage_policy"
```

Comment:
**suggestion** per discussion we want to name things once - `vsphere_storage_policy.storage_policy` is twice, should be changed to `vsphere_storage_policy.get` or something to that effect. We also probably want this to be a module, not a state, since it's just getting and not enforcing the state.

Reply:
As suggested in call, storae_policy was renamed to vsphere, and from there we can access several state properties, similar to esxi